src/main/java/com/odysseusinc/arachne/executionengine/service/impl/KerberosServiceImpl.java

```diff
@@ -24,26 +31,39 @@
 
     private static final Logger log = LoggerFactory.getLogger(KerberosService.class);
     public static final String LOG_FILE = "kinit_out.txt";
+    private Map<String, String> krbEnvProps = new HashMap<>();
```

Comment:
How is it going to work in multi-threaded env? At least, it should be ThreadLocal, but I'd prefer functional approach